django/contrib/admin/utils.py

```diff
@@ -482,6 +486,11 @@ def display_for_value(value, empty_value_display, boolean=False):
         return formats.number_format(value)
     elif isinstance(value, (list, tuple)):
         return ", ".join(str(v) for v in value)
+    elif isinstance(value, str) and not value.strip():
+        value = value.replace(" ", "\xa0")
```

Comment:
Do we need this generally for strings? How is something like `"test        "` displayed?